### PR TITLE
Add metrics to see what workflows are scheduled

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: investigator/investigator/common.py [hunk 1]
```diff
@@ -46,6 +46,18 @@ async def wait_for_limit(openshift: OpenShift, workflow_namespace: str):
         await sleep(Configuration.SLEEP_TIME)
         total_pending = openshift.workflow_manager.get_pending_workflows(workflow_namespace=workflow_namespace)
 
+async def schedule_kebechet_run_url(
+    openshift: OpenShift, repo: str, gitservice_repo_name: str, indexes: List[str]
+) -> int:
+    """Schedule all solvers."""
+    try:
+        await wait_for_limit(openshift, workflow_namespace=Configuration.THOTH_BACKEND_NAMESPACE)
+        is_scheduled = openshift.schedule_kebechet_run_url(repo, gitservice_repo_name)
```
Comment (pacospace) [inline investigator/investigator/common.py:55]: I cannot find `openshift.schedule_kebechet_run_url` this method in thoth-common @KPostOffice

Comment (pacospace) [reply investigator/investigator/common.py:55]: Never mind, we have the internal trigger feature that will create that method, for now we just update the database

Comment (pacospace) [reply investigator/investigator/common.py:55]: https://github.com/thoth-station/common/issues/1009

Comment (KPostOffice) [reply investigator/investigator/common.py:55]: ++

Comment (pacospace) [reply investigator/investigator/common.py]: Added a TODO so we can have this in meanwhile

**Pull request merged**